public/app/plugins/panel/timeseries/plugins/ExemplarsPlugin.tsx

```diff
@@ -61,8 +69,41 @@ export const ExemplarsPlugin: React.FC<ExemplarsPluginProps> = ({ exemplars, tim
     };
   }, []);
 
+  // Merge values from objects with same props
+  const getUniqueValuesFromLabels = (labels: Labels[]) => {
+    const labelToSet: { [index: string]: Set<string> } = {};
+
+    Object.values(labels).forEach((labelObject) => {
+      return Object.keys(labelObject).forEach((labelName) => {
+        const labelValue: string = labelObject[labelName];
+        if (typeof labelToSet[labelName] === 'undefined') {
+          labelToSet[labelName] = new Set<string>().add(labelValue);
+        } else {
+          labelToSet[labelName].add(labelValue);
+        }
+      });
+    });
```

Comment:
```suggestion
    const labelToSet: Record<string, Set<string>> = {};

    Object.values(labels).forEach((labelObject) => {
      return Object.keys(labelObject).forEach((labelName) => {
        const labelValue: string = labelObject[labelName];
        if (!labelToSet[labelName]) {
          labelToSet[labelName] = new Set<string>();
        }
        labelToSet[labelName].add(labelValue);
      });
    });
```

Reply:
Also, I do not understand why we need to return inside `forEach` in line 77. I think there is no need to return. Am I missing something?

Reply:
This function isn't even being called, I'm removing it now.